mafia/roles.py

```diff
@@ -119,7 +121,7 @@ def __str__(self) -> str:
 
 
 class Citizen(Role):
-    id = 0
+    id = None
```

Comment:
Since all of these for every role is the same as the Role class, they aren't really needed

Reply:
this resolves https://canary.discord.com/channels/840698427755069470/840698427755069475/847639392469778463

Reply:
wontfix